arrays.py

```diff
@@ -110,8 +110,6 @@
     "Queer people",
     "Debian Maintainers",
     "Debian Developers",
-    "Debian Developers, uploading",
-    "Debian Developers, non-uploading",
```

Comment:
I'd prefer to keep these :3

Reply:
Ok, I don't understand it but I've reverted as requested (I think - I am absolutely not a programmer and this git stuff is all Greek to me :P)